embulk-input-mysql/src/main/java/org/embulk/input/MySQLTimeZoneBuilder.java

```diff
@@ -0,0 +1,49 @@
+package org.embulk.input;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Locale;
+import java.util.TimeZone;
+
+public class MySQLTimeZoneBuilder
+{
+    private static final int ONE_HOUR_SEC = 3600;
+    private static final int ONE_MIN_SEC = 60;
+
+    public static TimeZone fromSystemTimeZone(Connection connection)
+        throws SQLException
+    {
+        //
+        // First, I used `@@system_time_zone`. but It return non Time Zone Abbreviations name on a specific platform.
+        // So, This method calculate GMT offset with query.
+        //
+        String query = "select TIME_TO_SEC(timediff(now(),utc_timestamp()));";
+        Statement stmt = connection.createStatement();
+        ResultSet rs = stmt.executeQuery(query);
```

Comment:
`Statement` and `ResultSet` are not closed.
They should be closed in `finally` clause.
<pre>
Statement stmt = ...
try {
    ...
} finally {
    stmt.close();
}
</pre>